src/js/pre-need/config/form.js

```diff
@@ -714,7 +698,10 @@ const formConfig = {
                       'view:applicantInfo': {
                         type: 'object',
                         properties: {
-                          name: _.omit('required', fullName),
+                          name: _.merge(_.omit('required', fullName), {
```

Comment:
Is this meant to be adding the maiden name? I don't think the object you're merging is quite right here.

Also, looks like you could use `nonRequiredFullName`.

Reply:
Thanks @jbalboni, good catch, this has been updated to make use of the updated schema.